apps/proxy-auth/src/app/otp/component/login/login.component.ts

```diff
@@ -70,6 +74,23 @@ export class LoginComponent extends BaseComponent implements OnInit {
                     this.resetPasswordForm.get('confirmPassword').updateValueAndValidity();
                 }
             });
+        this.componentStore.apiError$.subscribe((error) => {
+            this.apiError.next(error);
+        });
+    }
+    public backToLogin() {
+        this.step = 1;
```

Comment:
Chang step in a func, and perform operation step-dependent operation in that func
like `backToLogin` and `resetPassword` can be handled in one func 
and it can also be used in `otpData$` and `resetPassword$` subscription 
``` 
public changeStep(nextStep: number) {
        this.step = nextStep;
        this.apiError.next(null);
    }
```